runtime/default.go

```diff
@@ -224,18 +227,20 @@ func (r *runtime) run(events <-chan Event) {
 		case <-t.C:
 			// check running services
 			r.RLock()
-			for _, service := range r.services {
-				if !service.ShouldStart() {
-					continue
-				}
+			for _, sevices := range r.namespaces {
+				for _, service := range sevices {
+					if !service.ShouldStart() {
+						continue
+					}
 
-				// TODO: check service error
-				if logger.V(logger.DebugLevel, logger.DefaultLogger) {
-					logger.Debugf("Runtime starting %s", service.Name)
-				}
-				if err := service.Start(); err != nil {
+					// TODO: check service error
 					if logger.V(logger.DebugLevel, logger.DefaultLogger) {
-						logger.Debugf("Runtime error starting %s: %v", service.Name, err)
+						logger.Debugf("Runtime starting %s", service.Name)
```

Comment:
Might be nice to update the log statement to include the namespace that the service lives in